cav_msgs/msg/RouteEvent.msg

```diff
@@ -12,10 +12,10 @@ Header header
 
 uint8 event
 
-uint8 LOAD_ROUTE_FILES=0
+uint8 ROUTE_LOADED=0
 uint8 ROUTE_SELECTED=1
 uint8 ROUTE_STARTED=2
 uint8 ROUTE_COMPLETED=3
-uint8 LEFT_ROUTE=4
+uint8 ROUTE_DEPARTURE=4
```

Comment:
Since everything is past tense, let's update this to ROUTE_DEPARTED instead of ROUTE_DEPARTURE.